flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -969,6 +971,8 @@
 
 	public static final String DEFAULT_ZOOKEEPER_DIR_KEY = "/flink";
 
+	public static final String DEFAULT_ZOOKEEPER_NAMESPACE_KEY = "/default_ns";
```

Comment:
How about we change this to `default`? `ns` is pretty cryptic for most people anyways.